chasm/registry.go

```diff
@@ -81,6 +81,15 @@ func (r *Registry) ComponentFqnByID(id uint32) (string, bool) {
 	return fqn, ok
 }
 
+// ComponentByID returns the registrable component for a given archetype ID.
```

Comment:
plz also comment that those methods should not be used by chasm library developer. 

we probably need to create a separate interface for developer facing registry later.